spynnaker/pyNN/models/neuron/synaptic_manager.py

```diff
@@ -325,8 +325,9 @@ def get_sdram_usage_in_bytes(
 
         :param ~pacman.model.graphs.common.Slice post_vertex_slice:
             The slice of atoms to get the size of
-        :param ~pacman.model.graphs.application.ApplicationGraph \
-                application_graph: The application graph
+        :param application_graph: The application graph
+        :type application_graph: \
+            ~pacman.model.graphs.application.ApplicationGraph
```

Comment:
Seems pointless...

Reply:
pointless? when you take it as a arg...... confused

Reply:
Sorry, I meant the change seems pointless. I think Donal did quite a bit of work to merge the type and the param, but this separates them again.

Reply:
actually. its coz pycharm dislikes the variable name on the next line. it declaires it as not doced. whereas this way pycharm is happy. and its just as good for the docs